v1/components/FriendlySlugField.tsx

```diff
@@ -19,91 +19,94 @@ const makeFriendlyStr = (s: string): string => {
-}
+	return friendly;
+};
 
 const FriendlyURLField = () => {
-	const { contentItem, field, fieldValue } = useAgilityAppSDK()
+	const { contentItem, field, fieldValue } = useAgilityAppSDK();
 
-	const [currentTitle, setCurrentTitle] = useState<string>("")
-	const [CTAIcon, setCTAIcon] = useState<UnifiedIconName | undefined>(undefined)
-	const [CTALabel, setCTALabel] = useState<string | undefined>(undefined)
-	const [width, setWidth] = useState<number>(document.body.clientWidth)
+	const [currentTitle, setCurrentTitle] = useState<string>("");
+	const [CTAIcon, setCTAIcon] = useState<UnifiedIconName | undefined>(undefined);
+	const [CTALabel, setCTALabel] = useState<string | undefined>("Re-Generate Slug");
+	const [width, setWidth] = useState<number>(document.body.clientWidth);
 
 	const regenerateSlug = async (title: string) => {
-		const newVal = makeFriendlyStr(title)
-		contentItemMethods.setFieldValue({ name: field?.name, value: newVal })
-	}
-	const hasBeenSaved = !!!(contentItem && contentItem?.contentID < 0)
+		const newVal = makeFriendlyStr(title);
+		contentItemMethods.setFieldValue({ name: field?.name, value: newVal });
+	};
+	const hasBeenSaved = !!!(contentItem && contentItem?.contentID < 0);
 
 	const intializeTitleFieldListener = () => {
 		contentItemMethods.addFieldListener({
 			fieldName: "Title",
 			onChange: (t) => {
-				setCurrentTitle(t)
+				setCurrentTitle(t);
 				contentItemMethods?.getContentItem()?.then((ci) => {
 					//only regenerate the slug if the content item is new
-					const isNewContentItem = Boolean((ci?.contentID ?? -1) < 0)
+					const isNewContentItem = Boolean((ci?.contentID ?? -1) < 0);
 					if (isNewContentItem) {
-						regenerateSlug(t)
+						regenerateSlug(t);
 					}
-				})
+				});
 			}
-		})
-	}
+		});
+	};
 
 	const handleResizeListener: (isCleanup?: boolean) => void = (isCleanup) => {
-		const handleResize = () => setWidth(document.body.clientWidth)
+		const handleResize = () => setWidth(document.body.clientWidth);
 		if (isCleanup) {
-			window.removeEventListener("resize", handleResize)
-			return
+			window.removeEventListener("resize", handleResize);
+			return;
 		} else {
-			window.addEventListener("resize", handleResize)
-			return
+			window.addEventListener("resize", handleResize);
+			return;
 		}
-	}
+	};
 
 	useEffect(() => {
-		setHeight({ height: 50 })
-		intializeTitleFieldListener()
-		handleResizeListener()
+		setHeight({ height: 50 });
+		intializeTitleFieldListener();
+		handleResizeListener();
 		return () => {
-			handleResizeListener(true)
-		}
-	}, [])
+			handleResizeListener(true);
+		};
+	}, []);
 
 	useEffect(() => {
 		if (hasBeenSaved) {
-			setCTAIcon("RefreshIcon")
-			setCTALabel("Re-Generate Slug")
+			setCTAIcon("RefreshIcon");
+			setCTALabel("Re-Generate Slug");
 			if (width < 400) {
-				setCTALabel(undefined)
+				setCTALabel(undefined);
 			}
 		} else {
-			setCTAIcon(undefined)
-			setCTALabel(undefined)
+			setCTAIcon(undefined);

```

Comment:
Button label should only be undefined when the width < 400. Otherwise the field looks janky.